### ISSUE TITLE
FeeEstimation missing pattern match branch

### ISSUE THREAD

**Issue by zkao** [farcaster-project/farcaster-node#658]
```
10:20 <TheCharlatan/libera> I'm seeing "[2022-08-16T08:16:54Z ERROR farcaster_node::swapd::runtime] no request consumed with this predicate
10:20 <TheCharlatan/libera> [2022-08-16T08:16:54Z DEBUG farcaster_node::swapd::runtime] failed dispatching reveal:aliceparams, maybe sent beforeY
10:20 <TheCharlatan/libera> A bunch of times now.
10:21 <TheCharlatan/libera> I don't think it is an actual bug, but it is interesting that this is triggered at all.

10:32 <lederstrumpf[m]/libera> > I'm seeing "[2022-08-16T08:16:54Z ERROR farcaster_node::swapd::runtime] no request consumed with this predicate
10:32 <lederstrumpf[m]/libera> ditto

10:53 <zkao> > I don't think it is an actual bug, but it is interesting that this is triggered at all.
10:54 <zkao> thank you for reporting
10:55 <zkao> i forgot to write a pattern that catches FeeEstimation after Alice reveals its parameters, it seems
10:56 <zkao> the behavior is correct though, despite the message
```

**Comment by TheCharlatan** [comment]
Look at this again once #743 is done.

**Comment by TheCharlatan** [comment]
@h4sh3d did your bus refactor incidentally fix this? I'm not seeing this log appearing anymore.

**Comment by h4sh3d** [comment]
In #743 we removed the deferred request usage in commit/reveal, so yes this should have been a nice side effect of this refactor.

**Comment by TheCharlatan** [comment]
Ok, closing then.
